data/starts.txt

```diff
@@ -59,11 +70,130 @@ conversation "default intro"
+			`Representation in the Republic Parliament is based on population. That means that some individual "Paradise Worlds" have more representatives than entire regions of space like the Dirt Belt that are more sparsely settled.`
+		log "Factions" "Syndicate" `The Syndicate is a megacorporation, the largest employer in human space. People who cannot find steady work elsewhere flock to the Syndicate factory worlds, where they are almost guaranteed to find a job. Although the Syndicate is technically part of the Republic, Syndicate worlds are governed directly by the corporation.`
+			`The Syndicate is a central part of the Republic's economy, but they are also well known for selling shoddy products and for doing damage to the environment on the worlds they control. Their treatment of workers is questionable, and their opposition to organized labor is legendary. Some Syndicate factory towns have even been accused of human rights abuses.`
+		log "Factions" "Pirates" `In poorer and more remote star systems, where the Navy seldom patrols, pirate attacks on merchant ships are frequent. Pirates are also known to attack large, unguarded convoys of freighters even in more populated areas. Most pirate fleets come from lawless worlds on the outskirts of human space.`
+	scene "scene/icy opening"
+	`Delve is unlike any planet you've ever been on: the entire spaceport is built into a colossal mountain cave, one side exposed to the open air. The beams and buildings making up the port dwarf the pitiful structures of Antipode, yet feel distinct from the crushing urban landscapes of Earth.`
+	`	You hand the last of your credits to the captain that brought you here. Before she leaves, she gives you some advice. "Just a heads up: if you don't feel like dying early, head north after launching. South of here is nothing but uninhabited systems and pirates. You wouldn't last long in any ship you could afford." You thank her for the ride and advice before parting ways.`
+	`	You stop by the shipyard first to see what they have. You're noticed by a salesman who seems reluctant to approach you, perhaps due to the shabbiness of your clothes compared to those around you. You ask him what it would take to purchase a ship here. He mentions the necessity of a pilot's license before beginning his spiel about all the cheapest models, but you cut him short and say that's all you needed.`
+	`	The bank is luckily not very busy at the moment. One of the tellers eyes you up as you enter. "What can we help you with?" he asks with unhidden glee. You say how you're in need of a loan to buy a starship; 600,000 credits should be enough to cover the cost of the license and the ship. After handing over your Syndicate Account information, the teller comes back with a piece of paper. This whole process seems suspiciously streamlined to you.`
```

Comment:
Bit of phrasing cleanup, and couple other tweaks. In the Syndicate Business chain, financial transactions require a biometric and is overseen by Syndicated Systems, so added the biometric & "Systems" for some continuity.

```suggestion
	`	Luckily, the bank isn't very busy at the moment. One of the tellers sizes you up as you enter. "What can we help you with?" he asks with unrestrained glee. You say you're in need of a loan to buy a starship; 600,000 credits should be enough to cover the the license and the ship. After you hand over your Syndicated Systems Account information and scan your fingerprint, the teller returns with a piece of paper.`
```